README.md

```diff
@@ -11,149 +11,216 @@ with built-in validation, testing, sanitising and encryption.
-
-  <a href="https://youtu.be/8UQK-UcRezE"
-  alt="Try the Demo on Heroku!">
-    <img src="https://user-images.githubusercontent.com/194400/57781239-af4a0280-7721-11e9-8bf0-42bee626f58a.png"
-    alt="Van Gogh Fields of Wheat!">
-  </a>
+<!-- uncomment when service is working ...
+[![Inline docs](http://inch-ci.org/github/dwyl/fields.svg?branch=master&style=flat-square)](http://inch-ci.org/github/dwyl/fields)
 -->
 
 </div>
 
+# _Why_? 🤷
+
+We found ourselves repeating code
+for commonly used fields on each new Phoenix project/App.  
+We wanted a _much_ easier/faster way of building apps
+so we created a collection of pre-defined fields
+with built-in validation, sanitising and security.
+**`Fields`** makes defining Ecto Schemas faster
+and more precise.
+
+
+# _What_? 💭
+
+An Elixir package that helps you add popular custom types
+to your Phoenix/Ecto schemas so you can build apps faster!
+
+> **@dwyl** we are firm believers that personal data
+(_Personally Identifiable Information_ (PII)) should be encrypted "at rest"
+i.e. all "user" data should be encrypted _before_ being stored in the database.
+This project makes hashing, encryption and _decryption_ for secure data storage
+_much_ easier for everyone.
 
+# _Who_? 👥
 
-## Installation
+This module is for people building Elixir/Phoenix apps
+who want to ship _simpler_ more maintainable code.
 
-Add the `fields` package to your list of dependencies in `mix.exs`:
+> We've attempted to make **`Fields`**
+as **beginner-friendly** as possible. <br />
+If you get stuck using it or anything is unclear, please ask for
+[help!](https://github.com/dwyl/fields/issues)
+
+# _How_? ✅
+
+Start using **`Fields`** in your Phoenix App today with these 3 easy steps:
+
+
+## 1. Add the `fields` hex package to `deps` in `mix.exs` 📦
+
+Add the `fields` package to your list of dependencies in your `mix.exs` file:
 
 ```elixir
 def deps do
   [
-    {:fields, "~> 1.0.0"}
+    {:fields, "~> 2.1.0"}
   ]
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). 
-Once published, the docs can
-be found at [https://hexdocs.pm/fields](https://hexdocs.pm/fields).
+Once you have saved the `mix.exs` file,
+run **`mix deps.get`** in your terminal to download.
 
-## Usage
 
-Each field can be used in place of an Ecto type when defining your schema.
+## 2. Ensure you have the necessary environment variables 🔑
 
-```
-schema "users" do
-  field(:email, Fields.EmailEncrypted)
-  field(:address, Fields.Address)
-  field(:postcode, Fields.Postcode)
-  field(:password, Fields.Password)
+In order to use Encryption and Hashing,
+you will need to have environment variables
+defined for `ENCRYPTION_KEYS` and `SECRET_KEY_BASE` respectively.
 
-  timestamps()
-end
+```yaml
+export ENCRYPTION_KEYS='nMdayQpR0aoasLaq1g94FLba=,L+ZVX8iheoqgqb22mUpATmMDsvVGtafoAeb='
```

Comment:
I'm wondering if we need to have two keys in the `ENCRYPTION_KEYS` or if we should just have one to make the example simple?
Or we could add a link  to https://github.com/dwyl/phoenix-ecto-encryption-example#2-create-the-user-schema-database-table which mention key rotation?

Reply:
@SimonLab good point. we only need _one_ value in `ENCRYPTION_KEYS` for simplicity.

Reply:
The Background / Further Reading section mention already the phoenix-ecto-encryption-example
![image](https://user-images.githubusercontent.com/6057298/69949616-c8fa3e00-14e9-11ea-9d86-7c15f63e5067.png)

Reply:
I've updated the `.env_sample` and `README.md` to only have one Encryption Key. 👍